starters/next-drupal-starter/lib/getPreview.js

```diff
@@ -51,10 +51,18 @@ export async function getPreview(context, node, params) {
 					}`,
 					requestInit,
 				);
+				if (!fetchedPreviewData?.data?.id) {
+					throw new Error(
+						'Unable to fetch preview data. Drafts that have been saved are not preview-able and must be published again before preview is possible.',
```

Comment:
This message will be awkward regardless, but what about:

Unable to fetch preview data. Previewing drafts is currently not supported. As an alternative, either preview while editing, or preview a saved revision.

Reply:
Works for me! change inc.